emails/views.py

```diff
@@ -274,29 +281,12 @@ def _sns_message(message_json):
     if text_content:
         message_body['Text'] = {'Charset': 'UTF-8', 'Data': text_content}
 
-    relay_from_address, relay_from_display = _generate_relay_From(from_address)
-    formatted_from_address = str(
-        Address(relay_from_display, addr_spec=relay_from_address)
+    ses_client = get_ses_client()
```

Comment:
`get_ses_client()` could return `None`, should we check that ses_client is NOT `None` before calling `ses_send_email`?

Reply:
Hmm ... this actually makes me think of a bigger refactor. Should we move the ses client connection "up" into `EmailsConfig.ready()` ? That should hold the client in application memory so we don't have to connect each time.